src/components/DiagramMoney.tsx

```diff
@@ -1,18 +1,75 @@
 import * as React from 'react';
 import { useEffect, useState } from 'react';
+
 import money from '../assets/stats/money.png';
-import { Input, Text } from '@chakra-ui/react';
+import { Text, Flex, Center, Box, VStack, HStack } from '@chakra-ui/react';
+import { Bar, BarChart, XAxis, YAxis } from 'recharts';
+import dataJson from '../assets/MockData.json';
 
 const DiagramMoney = () => {
     const [count, setCount] = useState<string>('');
 
     useEffect(() => {}, []);
 
+    function moneyToList() {
+        const moneyList: number[] = [];
+        dataJson.EnergyMoneyMonthly.forEach((month) =>
+            moneyList.push(month.spent)
+        );
+        return moneyList;
+    }
+
+    function moneyBoxHeight() {
+        const boxList: JSX.Element[] = [];
+
+        moneyToList().forEach(
+            (item) =>
+                boxList.push(
+                    <Box className="moneyBar" height={item / 166.66}></Box>
+                ) //Prøve på noe modelo så lappene blir hele
+        );
+
+        return boxList;
```

Comment:
Her er også et godt eksempel for å bruke map, feks: `moneyToList().map((item) => <Box className="moneyBar" height={item / 166.66} />` 

Da slipper du å ha egen liste som du pusher til :D

Reply:
Should be good now :)